workers/unity/spatialos.SimulatedPlayer.worker.json

```diff
@@ -14,7 +14,47 @@
     },
     "component_delivery": {
       "default": "RELIABLE_ORDERED",
-      "checkoutAllInitially": true
```

Comment:
n00b question - what does this change? Are there any component not checked out initially in a new version?

Reply:
`checkoutAllInitially` means that we used to get updates for every component on checked out entities

now, we check out all components _except_ the `ClientMovement` one, because we only care about other clients' `ServerMovement` component to determine where they are

note that this is overridden for entities you have authority over - so clients will still get the `ClientMovement` component for themselves but not for other players near them

Reply:
Makes sense. Is there any way for us to keep the default checkoutAllInitially, but only have override to false for ClientMovement that we don't have authority over?
That would make this much easier to understand - now it is difficult to parse what field is missing from overrides

Reply:
haha I know what you mean - I wish we could do that but according to the [docs](https://docs.improbable.io/reference/13.4/shared/worker-configuration/bridge-config#components-to-check-out-initially) we can only define overrides if `checkoutAllInitially` is false (not providing it defaults to false)

Reply:
a neater way to do this on runtime v2 would be to purely rely on QBI and not set the entity interest radius, which would let us avoid fiddling with bridge configs like this

but for now this solution works across both runtimes